superbench/benchmarks/micro_benchmarks/gpu_copy_performance/CMakeLists.txt

```diff
@@ -25,6 +25,11 @@ else()
         # Convert cuda code to hip code in cpp
         execute_process(COMMAND hipify-perl -print-stats -o gpu_copy.cpp gpu_copy.cu WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/)
 
+        check_symbol_exists("hipDeviceMallocUncached" "hip/hip_runtime_api.h" HIP_UNCACHED_MEMORY)
```

Comment:
include(CheckSymbolExists)
![image](https://github.com/microsoft/superbenchmark/assets/37182275/905ed3be-93ea-4b0f-bb17-9137fc53518a)
is this expected?

Reply:
Yes, this is only available some certain ROCm version. Ref: https://github.com/ROCm/rccl/blob/develop/CMakeLists.txt#L572